src/hyp3_isce2/stripmapapp_alos.py

```diff
@@ -123,7 +123,7 @@ def run_stripmapapp(dostep: str = '', start: str = '', end: str = '', config_xml
         ValueError: If the step is not a valid step (see TOPSAPP_STEPS)
     """
     if not config_xml.exists():
-        raise IOError(f'The config file {config_xml} does not exist!')
+        raise OSError(f'The config file {config_xml} does not exist!')
```

Comment:
`IOError` is just a legacy/backwards compatible alias for `OSError` so `OSError` is preferred.